python/twilio-labels/whats_that.py

```diff
@@ -92,10 +110,28 @@ def construct_message(labels):
     return resp
 
 
-def get_labels(image, num_retries=3, max_results=3):
-    """Given an image, execute the label request"""
+def extract_sentiment(emotions):
+    """Extract the sentiment from the facial annotations"""
+    joy = [0, 0, 0]
+    sorrow = [0, 0, 0]
+    anger = [0, 0, 0]
+    surprise = [0, 0, 0]
+    odds = ['VERY_LIKELY', 'LIKELY', 'POSSIBLE']
+
+    # Loop through the emotions we're pulling and get the count
+    for i in range(len(odds)):
+        joy[i] = sum(f['joyLikelihood'] == odds[i] for f in emotions)
+        anger[i] = sum(f['angerLikelihood'] == odds[i] for f in emotions)
+        sorrow[i] = sum(f['sorrowLikelihood'] == odds[i] for f in emotions)
+        surprise[i] = sum(f['surpriseLikelihood'] == odds[i] for f in emotions)
+
+    return joy, anger, sorrow, surprise
 
+
+def get_labels(image, num_retries=3, max_labels=3, max_faces=10):
+    """Given an image, execute the label request"""
     labels = ""
+    face_annotations = ""
 
     # Set up the service that can access the API
     http = httplib2.Http()
```

Comment:
You should be able to simplify this bit -- see e.g. https://github.com/GoogleCloudPlatform/cloud-vision/blob/master/python/label/label.py#L46